functions-and-operators/json-functions/json-functions-modify.md

```diff
@@ -0,0 +1,346 @@
+| JSON_REPLACE('{"a": 41, "b": 62}','$.b',42,'$.c',43) |
++------------------------------------------------------+
+| {"a": 41, "b": 42}                                   |
++------------------------------------------------------+
+1 row in set (0.00 sec)
+```
+
+## [JSON_SET()](https://dev.mysql.com/doc/refman/8.0/en/json-modification-functions.html#function_json-set)
+
+The `JSON_SET(json_doc, path, value [,path, value] ...)` function inserts or updates data in a JSON document and returns the result.
+
+This function takes arguments in pairs, where each pair is a `path` and a `value`.
+
+Examples:
+
+In the following example, you can update the `$.version` from `1.1` to `1.2`.
+
+```sql
+SELECT JSON_SET('{"version": 1.1, "name": "example"}','$.version',1.2);
+```
+
+```
++-----------------------------------------------------------------+
+| JSON_SET('{"version": 1.1, "name": "example"}','$.version',1.2) |
++-----------------------------------------------------------------+
+| {"name": "example", "version": 1.2}                             |
++-----------------------------------------------------------------+
+1 row in set (0.00 sec)
+```
+
+In the following example, you update the `$.version` from `1.1` to `1.2`. And you update `$.branch`, which is not present before, to `main`.
+
+```sql
+SELECT JSON_SET('{"version": 1.1, "name": "example"}','$.version',1.2,'$.branch', "main");
+```
+
+```
++------------------------------------------------------------------------------------+
+| JSON_SET('{"version": 1.1, "name": "example"}','$.version',1.2,'$.branch', "main") |
++------------------------------------------------------------------------------------+
+| {"branch": "main", "name": "example", "version": 1.2}                              |
++------------------------------------------------------------------------------------+
+1 row in set (0.00 sec)
+```
+
+## [JSON_UNQUOTE()](https://dev.mysql.com/doc/refman/8.0/en/json-modification-functions.html#function_json-unquote)
+
+The `JSON_UNQUOTE(json)` function unquotes a JSON value and returns the result as a string. This is the opposite of the [`JSON_QUOTE()`](/functions-and-operators/json-functions/json-functions-create.md#json_quote) function.
+
+Examples:
+
+In the example `"foo"` is unquoted to `foo`.
+
+```sql
+SELECT JSON_UNQUOTE('"foo"');
+```
+
+```
++-----------------------+
+| JSON_UNQUOTE('"foo"') |
++-----------------------+
+| foo                   |
++-----------------------+
+1 row in set (0.00 sec)
+```
+
+This function is often used together with [`JSON_EXTRACT()`](/functions-and-operators/json-functions/json-functions-search.md#json_extract). In the following example, you first extract a value, which is quoted, and in the second example you combine this to unquote the value. Note that instead of `JSON_UNQUOTE(JSON_EXTRACT(...))`, you can use the [`->>`](/functions-and-operators/json-functions/json-functions-search.md#--1) shorthand.
```

Comment:
```suggestion
This function is often used together with [`JSON_EXTRACT()`](/functions-and-operators/json-functions/json-functions-search.md#json_extract). For the following examples, you can extract a JSON value with quotes in the first example and then use the two functions together to unquote the value in the second example. Note that instead of `JSON_UNQUOTE(JSON_EXTRACT(...))`, you can use the [`->>`](/functions-and-operators/json-functions/json-functions-search.md#--1) operator.
```